Neuron/Services/Ethereum/EthereumTxSender.swift

```diff
@@ -49,6 +49,10 @@ class EthereumTxSender {
         transaction.transaction.value = value
 
         let result = try transaction.sendPromise(password: password).wait()
+
+        let sentTransaction = SentTransaction(tokenType: .ethereum, from: from, sendingResult: result)
```

Comment:
Do not inject `SentTransaction` logic into TxSender. TxSender should not have knowledge of that.